db/schema.rb

```diff
@@ -11,21 +11,29 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-
-ActiveRecord::Schema.define(version: 20161123124500) do
+ActiveRecord::Schema.define(version: 20161129120651) do
 
   create_table "application_letters", force: :cascade do |t|
     t.string   "motivation"
-    t.integer  "user_id",    null: false
-    t.integer  "event_id",   null: false
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.boolean  "status"
+    t.integer  "user_id",                null: false
+    t.integer  "event_id",               null: false
+    t.datetime "created_at",             null: false
+    t.datetime "updated_at",             null: false
+    t.integer  "status",     default: 2, null: false
   end
 
   add_index "application_letters", ["event_id"], name: "index_application_letters_on_event_id"
   add_index "application_letters", ["user_id"], name: "index_application_letters_on_user_id"
 
+  create_table "application_notes", force: :cascade do |t|
+    t.text     "note"
+    t.integer  "application_letter_id"
+    t.datetime "created_at",            null: false
+    t.datetime "updated_at",            null: false
+  end
+
+  add_index "application_notes", ["application_letter_id"], name: "index_application_notes_on_application_letter_id"
+
```

Comment:
Keeping it as it is (if it is the newest version available) is a good idea. However, you need to merge dev into this branch again (conflict in this exact file) and should probably reverse & run the migrations again, just to be sure :)